sos/report/plugins/microcloud.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2024 Alan Baghumian <alan.baghumian@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class MicroCloud(Plugin, UbuntuPlugin):
+    """The MicroCloud plugin collects the current status of the microcloud
+    snap.
+
+    It will collect journald logs as well as output from various microcloud
+    commands.
+    """
+
+    short_desc = 'MicroCloud Snap'
+    plugin_name = "microcloud"
+    profiles = ('container',)
+
+    packages = ('microcloud',)
+
+    microcloud_cmd = "microcloud"
+
+    def setup(self):
+        self.add_journal(units="snap.microcloud.*")
```

Comment:
do we need to check for service status as well?

Reply:
Nah, the systemd module is already collecting it, so no need for a duplicated efforts